src/_vars.scss

```diff
@@ -7,6 +7,7 @@
 
 // For use in a .js file, see `src/theme.js`.
 
+// DEPRECATED by theme palette update, colors here should be removed eventually in the future!
 $primary-blue: #014983;
```

Comment:
Please find all uses of this hex code, at least in this pull request, and replace them with the appropriate name.  Do the same for all the rest of the hex codes below.  (Exception: if there are two logically distinct names and uses for the same hex code, presumably used in different contexts, that's probably OK.)

Reply:
See below comment for why outdated